packages/fast-components-react-base/src/viewport-positioner/viewport-positioner.tsx

```diff
@@ -18,6 +19,8 @@ import {
     ViewportPositionerUnhandledProps,
     ViewportPositionerVerticalPosition,
 } from "./viewport-positioner.props";
+import { ThemeProvider } from "emotion-theming";
```

Comment:
this doesn't appear to get used, it's also not part of our package dependencies.

Reply:
Thanks for catching.  VS likely inserted that trying to be helpful.